src/gui/tray/UserModel.cpp

```diff
@@ -417,12 +417,16 @@ ActivityListModel *User::getActivityModel()
 
 void User::openLocalFolder()
 {
+    const auto folder = getFolder();
+
+    if (folder != nullptr) {
```

Comment:
nitpick: I'd go for `if (folder) {`

Reply:
In pre-C++11 times, I would have agreed. However, since `nullptr` has its own type, I prefer to be explicit and say `if (folder != nullptr)`; as I would also prefer `if (someInt != 0)` over `if (someInt)`. The only cases where I would not use any further operators is for checking bools, like `if (someBool)` and `if (!someBool)`.

Also, I have checked before, on master:
```
$  git grep '[!=]= nullptr'  | wc -l
41
```

So I can, of course, change it if you insist on it, but I personally consider the explicit comparison to be better.

Reply:
I guess we'll agree to disagree on that one. As I said it's a nitpick, I won't fight for it. ;-)

For the record: I agree with you on types like like int where it's clearly due to compatibility with C. Regarding pointers it's more of a conscious choice from the people writing the standard IMHO. To me a sign of that is the implicit operator bool added to all the smart pointer classes.